src/match/views.py

```diff
@@ -36,6 +36,34 @@ def post(self, request):
         return CreateMatchController(data, self.serializer_class).process()
 
 
+class MultipleMatchesView(generics.GenericAPIView):
+    serializer_class = BothUsersMatchSerializer
+    permission_classes = api_settings.ADMIN_PERMISSIONS
+
+    def post(self, request):
+        """Create multiple matches."""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
+        match_ids_list = data.get("matches")
+        if not match_ids_list:
+            return failure_response(
+                "POST body is misformatted", status=status.HTTP_400_BAD_REQUEST
+            )
+        for match_ids in match_ids_list:
+            new_match_response = CreateMatchController(
+                {"ids": match_ids}, self.serializer_class
+            ).process()
+            if new_match_response.status_code not in [
```

Comment:
This will work but wondering if there is a way of converting the status code to a number and checking if it's >= 400

Reply:
Looked into this and i see that we can use `is_client_error(code)` and `is_server_error(code)` - in the interest of time I'm going to take a look at this after we have created our first round of matches